networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ The following advisory is available for the AWS Load Balancer Operator version 1
 
 * link:https://access.redhat.com/errata/RHEA-2023:1954[RHEA-2023:1954 Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]
 
+[IMPORTANT]
+====
+The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Comment:
Should there be an (ALB) after AWS Load Balancer since the appreciation is used in the second sentence?
```suggestion
The AWS Load Balancer (ALB) Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Reply:
fixed!